src/lib/y2partitioner/device_graphs.rb

```diff
@@ -0,0 +1,47 @@
+require "singleton"
+
+module Y2Partitioner
+  # Object that holds and manipulates of storage device graphs.
+  class DeviceGraphs
```

Comment:
For reference, the other copy of this file is at https://github.com/yast/yast-partitioner/pull/5/files#diff-92f72368516441ecff55e03c66454cee